pylint/test/functional/consider_using_any_all.txt

```diff
@@ -0,0 +1,4 @@
+consider-using-any-all:12::Consider using 'any' for variables 'a, b, c' instead of using 'or' operator
```

Comment:
Can we instead recommend the actual code, as in:
```Consider using `any((a, b, c))` instead of using the `or` operator```

Reply:
yup this seems better, will try to figure out, how much effort it will require.